journal/updated/05-chapter-4.adoc

```diff
@@ -39,205 +39,205 @@ Bristol, Fourth month 30th, 1763.
-being assured,
-that all those who are '`faithful unto death,`' shall receive '`a crown of life.`'
+I think I may say, "`I know that peace,`" and beg that I may be preserved to the end,
+being assured
+that all those who are '`faithful unto death`' shall receive '`a crown of life.`'
 
 --
 
-I was enabled to leave Bristol with a degree of satisfaction, on the 16th of fifth month,
-accompanied by William Bragg and William Bush:--Robert
-Peters also going to Hotwells to see his worthy wife,
+I was enabled to leave Bristol with a degree of satisfaction on the 16th of fifth month,
+accompanied by William Bragg and William Bush. Robert
+Peters was also going to Hotwells to see his worthy wife,
 who seems to be near her change, having fought a good fight,
 and no doubt will finish her course with joy.
-Mary Winter of Stourbridge, accompanied me into Wales.
-We rode to New Passage, and with difficulty got safe over to the county of Monmouth.
+Mary Winter of Stourbridge accompanied me into Wales.
+We rode to New Passage and with difficulty got over safely to the county of Monmouth.
 
 Our first meeting was at Sheernewton, to which came several of the neighbours,
 and we had a satisfactory opportunity.
-We then proceeded to Pontypool, a poor place as to the situation of our professors;
-but we had cause to be thankful to our Lord who sent us,
+We then proceeded to Pontypool, a poor place as to the state of our society,
+but we had cause to be thankful to our Lord who sent us
 and was pleased to show unto us the states of the people,
 with whom we had a meeting on the 18th. We then visited a few families in the town,
-one of which, or rather the father thereof had done wickedly, but we knew it not,
-yet were led to pay them a visit;
-and soon after entering the house I had to believe that something was wrong;
+one of which (or rather the father thereof) had done wickedly, but we knew it not,
+yet were led to pay them a visit.
+Soon after entering the house I was made to believe that something was wrong;
 after a time of silence I found an engagement to speak,
 there being present two children of this person both grown up,
 himself and his second wife.
-At first the old man looked very stiff; but though an entire stranger,
+At first the old man looked very stiff, but though an entire stranger,
 I was led to speak very closely to his state and to mourn with the children,
 desiring them not to bring forth the fruits of Sodom, nor the grapes of Gomorrah,
 although their teeth had been set on edge by their parents;
 neither to be worshippers of idols, nor in anyway to defile themselves; telling them,
 that the Lord was of purer eyes than to behold iniquity with approbation,
-neither should the sinner go unpunished.
-Yet, although I was thus led, I had to speak of the beautiful situation of those,
+and the sinner would not go unpunished.
+I also felt led to speak of the beautiful situation of those
```

Comment:
I think we need to keep the first part of this sentence, because she is describing a contrast... saying something like, Even though I had some hard words to offer, I also had some encouraging ones.  So maybe: "Yet, though I was led to speak in this way, I was also made to speak of the beautiful...."